R/get_equations.R

```diff
@@ -0,0 +1,64 @@
+#' Generate all footnote equation options
+#' @description
+#'
+#' Generate data frame of generic footnote equations to append to parameter tables.
+#'
+#'
+#' @param .ci specify 90 or 95 percent confidence interval (default 95%)
+#' @param .zscore z-score for the specified confidence interval. Only needed for confidence intervals that are NOT 90 or 95 percent
+#'
+#' @examples
+#' #Using output from `format_param_table` (defineOut),
+#' library(dplyr)
+#' paramEst <- utils::read.csv(system.file("model/nonmem/param_est.csv", package = "pmparams"))
+#' paramKey <-  system.file("model/nonmem/pk-parameter-key-new.yaml", package = "pmparams")
+#' defineOut <- define_param_table(.estimates = paramEst, .key = paramKey, .ci = 95, .zscore = NULL)
+#' data <- format_param_table(.df = defineOut)
+#'
+#' #To make random effects table and add relevant footnotes:
+#' footnotes = get_equations()
+#' table <- make_pmtable(.df = data, .pmtype = "random") %>%
+#' pmtables::st_notes(footnotes$ci, footnotes$cv) %>%
+#' pmtables::st_notes_str() %>%
+#' pmtables::st_notes(footnotes$cvOmegaEq, footnotes$cvSigmaEq)
+#'
+#' @export
+get_equations <- function(.ci = 95, .zscore = NULL){
+
+  .validated_zscore <-
+    if (is.null(.zscore)){
+      if (.ci == 95){
+        1.64
+      } else if(.ci == 90){
+        1.96
+      } else {
+        stop("Z-score (.zscore) must be supplied when CI is not 90 or 95")
+      }
+    } else {
+      if (.ci == 95 & .zscore != 1.64){
+        zscore_ci_msg(.ci = 95, .zscore= 1.64)
+        1.64
+      } else if(.ci == 90 & .zscore != 1.96){
+        zscore_ci_msg(.ci = 90, .zscore= 1.96)
+        1.96
+      } else {
+        .zscore
+      }
+    }
+
+
+  list(
+    ci   = "CI: confidence intervals",
+    corr = "Corr: Correlation coefficient",
```

Comment:
lower case c on correlation
```suggestion
    corr = "Corr: correlation coefficient",
```